src/admin/getting_started_checks/design_removeDefaultBannersListingDetailsPage.php

```diff
@@ -50,18 +50,17 @@ public function isComplete()
         if (geoMaster::is('auctions')) {
             $name = 'listing_auction.tpl';
             $custom = file_get_contents(geoTemplate::getFilePath(geoTemplate::MAIN_PAGE, '', $name));
-            if ((strpos($custom, '<a href="#"><img src="{external file=\'images/banners/banner1_160w.jpg\'}" alt="" /></a>') !== false)) {
+            if (strpos($custom, 'sample_300x100.jpg') || strpos($custom, '1and1_300x100.jpg')) {
                 return false;
             }
         }
         if (geoMaster::is('classifieds')) {
             $name = 'listing_classified.tpl';
             $custom = file_get_contents(geoTemplate::getFilePath(geoTemplate::MAIN_PAGE, '', $name));
-            if ((strpos($custom, '<a href="#"><img src="{external file=\'../images/banners/banner1_160w.jpg\'}" alt="" /></a>') !== false)) {
+            if (strpos($custom, 'sample_300x100.jpg') || strpos($custom, '1and1_300x100.jpg')) {
```

Comment:
Again same here, the one it was checking for before is different than the one you are checking now...

Reply:
see above.